src/biogeochem/CNNStateUpdate1Mod.F90

```diff
@@ -98,13 +116,13 @@ subroutine NStateUpdate1(num_soilc, filter_soilc, num_soilp, filter_soilp, &
     integer                                 , intent(in)    :: filter_soilc(:) ! filter for soil columns
     integer                                 , intent(in)    :: num_soilp       ! number of soil patches in filter
     integer                                 , intent(in)    :: filter_soilp(:) ! filter for soil patches
-    type(cnveg_nitrogenflux_type)           , intent(in)    :: cnveg_nitrogenflux_inst
+    type(cnveg_nitrogenflux_type)           , intent(inout) :: cnveg_nitrogenflux_inst
     type(cnveg_nitrogenstate_type)          , intent(inout) :: cnveg_nitrogenstate_inst
     type(soilbiogeochem_nitrogenflux_type)  , intent(inout) :: soilbiogeochem_nitrogenflux_inst
     !
     ! !LOCAL VARIABLES:
     integer :: c,p,j,l,g,k ! indices
-    integer :: fp,fc     ! lake filter indices
+    integer :: fp,fc,nstep   ! lake filter indices
```

Comment:
Why is nstep declared in this line? is nstep lake filter index? I think no! So please correct it and stay consistent with the CTSM code.

Reply:
nstep is deleted.